extension/src/popup/basics/PopupWrapper/styles.scss

```diff
@@ -0,0 +1,5 @@
+.PopupWrapper {
+  max-height: 512px; // screen height minus nav height
```

Comment:
💭  we could probably just use the `--popup--height` var you added in `:root`?

Reply:
Good call, that's better